debezium-connector-mysql/src/test/java/io/debezium/connector/mysql/MySqlAntlrDdlParserTest.java

```diff
@@ -617,11 +617,76 @@ public void parseAlterEnumColumnWithNewCharacterSet() {
+                        "    `created` datetime DEFAULT CURRENT_TIMESTAMP,\n" +
+                        "    `modified` datetime DEFAULT CURRENT_TIMESTAMP ON UPDATE CURRENT_TIMESTAMP,\n" +
+                        "    PRIMARY KEY (`id`)\n" +
+                        ");\n" +
+                        "\n" +
+                        "ALTER TABLE `test_stations_11`\n" +
+                        "    MODIFY COLUMN `type` ENUM('station', 'post_office', 'plane', 'ahihi_dongok', 'now', 'a,b', 'c,\\'b')\n" +
+                        "    CHARACTER SET 'utf8' COLLATE 'utf8_unicode_ci' NOT NULL DEFAULT 'station';\n";
+
+        parser.parse(ddl, tables);
+        assertThat(tables.size()).isEqualTo(1);
+        assertThat(tables.forTable(new TableId(null, null, "test_stations_11"))).isNotNull();
+    }
+
+    @Override
+    @Test
+    @FixFor("DBZ-1226")
+    public void shouldParseEnumOptions() {
+        // NOTE: We've excluded the ENUM() use cases with empty values based that is not valid
+        // THe parser adheres to MySQL's grammar and expects a minimum of one option supplied in the data type definition.
+        assertParseEnumAndSetOptions("ENUM('a','b','c')", "a,b,c");
+        assertParseEnumAndSetOptions("enum('a','b','c')", "a,b,c");
+        assertParseEnumAndSetOptions("ENUM('a','multi','multi with () paren', 'other')", "a,multi,multi with () paren,other");
+        assertParseEnumAndSetOptions("ENUM('a')", "a");
+        assertParseEnumAndSetOptions("ENUM ('a','b','c') CHARACTER SET utf8", "a,b,c");
+        assertParseEnumAndSetOptions("ENUM ('a') CHARACTER SET utf8", "a");
+    }
+
+    @Override
+    @Test
+    @FixFor("DBZ-1226")
+    public void shouldParseEscapedEnumOptions() {
+        assertParseEnumAndSetOptions("ENUM('a''','b','c')", "a',b,c");
+        assertParseEnumAndSetOptions("ENUM('a\\'','b','c')", "a',b,c");
+        assertParseEnumAndSetOptions("ENUM(\"a\\\"\",'b','c')", "a\\\",b,c");
+        assertParseEnumAndSetOptions("ENUM(\"a\"\"\",'b','c')", "a\"\",b,c");
+    }
+
+    @Override
+    @Test
+    @FixFor("DBZ-1226")
+    public void shouldParseSetOptions() {
+        // NOTE: We've excluded the SET() use cases with empty values based that is not valid
+        // THe parser adheres to MySQL's grammar and expects a minimum of one option supplied in the data type definition.
+        assertParseEnumAndSetOptions("SET('a','b','c')", "a,b,c");
+        assertParseEnumAndSetOptions("set('a','b','c')", "a,b,c");
+        assertParseEnumAndSetOptions("SET('a','multi','multi with () paren', 'other')", "a,multi,multi with () paren,other");
+        assertParseEnumAndSetOptions("SET('a')", "a");
+        assertParseEnumAndSetOptions("SET ('a','b','c') CHARACTER SET utf8", "a,b,c");
+        assertParseEnumAndSetOptions("SET ('a') CHARACTER SET utf8", "a");
+    }
+
     @Override
     protected void assertParseEnumAndSetOptions(String typeExpression, String optionString) {
```

Comment:
@Naros, ha, already merged this, but one suggestion actually: what about defining this as var-args: 

    assertParseEnumAndSetOptions(String typeExpression, String... expected options)

Would avoid the join later on and feel a bit cleaner. A comment on the mechanics of the method might be nice as well.